lib/lita/handlers/google.rb

```diff
@@ -48,6 +56,21 @@ def search(response)
           )
         end
       end
+      private
+
+      def check_for_excluded_domains(data)
+        #reject all exlcuded domains
+        data["responseData"]["results"].find do |response|
+          uri = URI.parse(response["unescapedUrl"])
+          if uri && uri.host
```

Comment:
What doesn't work? It's the only thing this method needs to do. The body of a `Enumerable#find` block should return a boolean for whether or not the current iteration satisfies the condition.